app/pages/docs/file-structure.mdx

```diff
@@ -90,47 +90,48 @@ sidebar_label: File Structure
 
 #### `app`
 
-Contains all your core application code.
+Bevat alle core applicatiecode.
 
-- The file structure nested inside `app/` can be anything you want, but we recommend the following:
-  - First layer folders to be a "domain context", like `core/`, `projects/`, `users/`, `billing/`, etc.
-  - All other folders like `components/` and `hooks/` go inside one of the above domain context folders
-- Special Folder Names
-  - Can exist at any level of the hierarchy inside `app`.
-  - `pages/` expose a React component at a URL. Follows the same semantics as the Next.js `pages/` folder.
-  - `api/` expose a Node.js request handler at URL. Same semantics as Next.js `pages/api/` folder.
-  - `queries/` and `mutations/` are for your Blitz queries and mutations. Each query and mutation is exposed at a URL corresponding to its file path.
+- De bestandsstructuur binnen `app/` is vrijblijvend, maar we raden het volgende aan:
+  - De eerste laag mappen moeten een "domain context" zijn, zoals `core/`, `projects/`, `users/`, `billing/`, etc.
+  - Alle andere mappen zoals `components/` en `hooks/` gaan in één van de bovenstaande domain context mappen.
+- Speciale mapnamen
+  - Kunnen op elk niveau van de hiërarchie binnen `app` bestaan.
+  - `pages/` weergeeft een React component op een URL. Volgt dezelfde semantiek als de Next.js `pages/api/` map.
+  - `api/` geeft een Node.js request handler vrij op de URL. Volgt dezelfde semantiek als de Next.js `pages/api/` map.
```

Comment:
```suggestion
  - `api/` stel een Node.js request handler ter beschikking op een URL. Volgt dezelfde semantiek als de Next.js `pages/api/` map.
```

Reply:
Ik twijfelde hier ook over die de of een. Maar `pages` bijvoorbeeld is geen URL maar gewoon een map. Daarom had ik daar 'een' maar `api` is wel een deel van de URL bijv  `test.com/api/example` dus ik interpreteerde daarom het verschil tussen de twee in de Engelse versie.